docs/quickstart.md

```diff
@@ -0,0 +1,187 @@
+
+The use of pure and simple mathematics aims to answer the question "given the information I have, where and how fast do I move right this moment?"
+
+## Summary ##
+
+This toolkit is a framework for the [Godot engine](https://godotengine.org/). It takes a lot of inspiration from the excellent [GDX-AI](https://github.com/libgdx/gdx-ai) framework for the [LibGDX](https://libgdx.badlogicgames.com/) java-based framework. Every class in the toolkit is based on Godot's [Reference](https://docs.godotengine.org/en/latest/classes/class_reference.html) type. There is no need to have a complex scene tree; everything that has to do with the AI's movement can be contained inside movement oriented classes.
+
+As a short overview, a character is represented by a steering agent; it stores its position, orientation, maximum speeds and current velocity. A steering behavior is associated with a steering agent and it calculates a linear and/or angular change in speed from the information that is available. The coder then applies that acceleration in whatever ways is appropriate to the character to change its velocity.
+
+## More information and resources ##
+
+- [Understanding Steering Behaviors](https://gamedevelopment.tutsplus.com/series/understanding-steering-behaviors--gamedev-12732): Breakdowns of various behaviors by Fernando Bevilacqua with graphics and in-depth explanations.
+- [GDX-AI Wiki](https://github.com/libgdx/gdx-ai/wiki/Steering-Behaviors): Descriptions of how LibGDX's AI submodule uses steering behaviors with a few graphics. Since this toolkit uses it for inspiration, there will be some similarities.
+- [RedBlobGames](https://www.redblobgames.com/) - An excellent resources for complex pathfinding like A*, graph theory, and other algorithms that are game-development related. Steering behaviors are not covered, but for anyone looking to study and bulk up on their algorithms, this is a great place.
+
+## Example usage ##
+
+The fastest way to get started is to look at an explained sample class that makes use of the toolkit.
+
+The goal of this class is to represent an agent that chases the player and predicts where the player *will* be, but maintains a distance from him. When its health is low, it will flee from the player directly. The agent will keep facing the player while it is chasing it, but will look where it's going while it is fleeing. Our game will be in 2D, assumed to be a top-down spaceship game.
+
+You can see the demo in action by opening the `demos/QuickStartDemo.tscn` file in Godot. The result is an agent that approaches the player and hovers near him until it is shot enough times, at which point it will try to flee directly.
+
+More details about how the various steering behaviors function can be found in the [Reference](./reference.md) document.
+
+```ruby
+extends KinematicBody2D
+
+
+export var maximum_speed := 450.0 # Maximum possible linear velocity
+export var maximum_acceleration := 50.0 # Maximum change in linear velocity

```

Comment:
I don't know if we discussed it already but I'd recommend having names the other way around: `acceleration_max`, `speed_max`. The most important keyword first. It's the guideline the blender devs use, and as codebases grow, I find it leads to cleaner code. For instance, from the sequencer API:

```py
sequence.frame_start
sequence.frame_end
sequence.frame_duration
sequence.frame_final_start
...
```

Reply:
We hadn't until now, but I can fold it in throughout the codebase.

Reply:
It's more of a detail, but a matter of consistency. Also, at times, it's not clear which word should come first. It's more of a general note.